evaluation/promptfooconfig.yaml

```diff
@@ -0,0 +1,15 @@
+# This configuration compares LLM output of 2 prompts x 2 GPT models across 3 test cases.
```

Comment:
Is this comment right? I can't see where the 2 GPT models/3 tests cases are defined.

Reply:
Oops, that's a left over line from when the file was autogenerated. That's what the default generated script does. No GPT in this one, it just uses the python script (which uses Gemini)